shared/src/cell_set.rs

```diff
@@ -1,15 +1,49 @@
+use ckb_core::block::Block;
+use ckb_core::cell::{CellProvider, CellStatus};
 use ckb_core::transaction::OutPoint;
 use ckb_core::transaction_meta::TransactionMeta;
-use fnv::FnvHashMap;
+use fnv::{FnvHashMap, FnvHashSet};
 use numext_fixed_hash::H256;
 use serde_derive::{Deserialize, Serialize};
 
 #[derive(Default, Debug, Clone)]
 pub struct CellSetDiff {
```

Comment:
It's better to add some tests for `CellSetDiff`.

Reply:
added some basic tests, it's covered in https://github.com/nervosnetwork/ckb/pull/343/commits/22355a19b0897514610839c118b1c10812e4b03b